tests/artifactoryvirtualrepository_test.go

```diff
@@ -673,3 +675,25 @@ func virtualGenericTest(t *testing.T) {
 	assert.NoError(t, err, "Failed to update "+repoKey)
 	validateRepoConfig(t, repoKey, gvp)
 }
+
+func getVirtualRepoDetailsTest(t *testing.T) {
+	// Create Repo
+	repoKey := generateVirtualRepoKey("go")
+	gvp := services.NewGoVirtualRepositoryParams()
+	gvp.Key = repoKey
+	gvp.RepoLayoutRef = "simple-default"
+	gvp.Description = "Generic Repo for jfrog-client-go virtual-repository-test"
```

Comment:
Let's remove the "Generic" from the description, because we're not creating a generic repo here.